public/core.d.ts

```diff
@@ -3838,6 +3838,40 @@ declare namespace JMap {
      **/
     function clearFlashingLocations(): void
 
+    /**
+     * **JMap.Map.displayLayerExtent**
+     *
+     * Displays the layer extent of the given layer Id
+     *
+     * The layer extent will disapear shortly after a pan or a zoom
+     *
+     * @param layerId a layer Id
+     * @param options (see example)
+     * @example ```ts
+     *
+     * // Move to the layer 4 extent and display it for a few second
+     * JMap.Map.displayLayerExtent(4, {panAndZoom: true})
+     * ```
+     **/
+    function displayLayerExtent(layerId: JId, options?: JMapDisplayLayerExtentParams): void
+
+    /**
+     * **JMap.Map.clearDisplayedLayerExtent**
+     *
+     * Clears the currently displayed layer extent
+     *
+     * @param layerId a layer Id
+     * @example ```ts
+     *
+     * // display the layer's extent of layer 4 (as long as the user don't pan or zoom)
```

Comment:
plutot que `as long as the user don't pan or zoom` je mettrais plutôt `without panning and zooming`